### PR TITLE
Create admin config for payment 7ELEVEN

### PR DESCRIPTION
**Pull request by andykim**
## Description

Create admin config for payment 7ELEVEN

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Cosmetic change (text changing or color changing)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code (if necessary), particularly in hard-to-understand areas
- [ ] My changes generate no new vulnerabilities
- [ ] I have added tests that prove my fix is effective or that my feature works

## Screenshot

Screenshot from 2022-01-28 09-27-52

Screenshot from 2022-01-28 10-27-23
Screenshot from 2022-01-28 10-27-14

### CODE REVIEW

**Review by yanoandri — APPROVED** [review]

File: Xendit/M2Invoice/etc/config.xml [hunk 1]
```diff
@@ -232,6 +232,32 @@
             </shopeepay>
 
             <!-- PH Channels -->
+            <seven_eleven>
+                <active>1</active>
+                <payment_action>initialize</payment_action>
+                <model>Xendit\M2Invoice\Model\Payment\SevenEleven</model>
+                <title>7 Eleven</title>
```
Comment (IreneGohtami) [inline Xendit/M2Invoice/etc/config.xml:239]: let's name it: `7-Eleven`

**Review by IreneGohtami — APPROVED** [review]
minor comment, else lgtm

**Review by kevinalfianto — APPROVED** [review]

File: Xendit/M2Invoice/etc/config.xml (hunk 1 above)
Comment (andykim) [reply Xendit/M2Invoice/etc/config.xml:239]: ah, let me change

Comment (andykim) [reply Xendit/M2Invoice/etc/config.xml]: Hi @IreneGohtami . I changed the name to `7-Eleven`

Screenshot from 2022-01-28 10-41-33
Screenshot from 2022-01-28 10-41-29
Screenshot from 2022-01-28 10-41-26

**Pull request merged**